libs/ui/src/components/NeoModal/NeoModal.scss

```
@@ -4,14 +4,15 @@
 @import '@oruga-ui/oruga-next/src/scss/utilities/_variables.scss';
 @import '@oruga-ui/oruga-next/src/scss/utilities/_animations.scss';
 @import '@oruga-ui/oruga-next/src/scss/utilities/_helpers.scss';
+@import '@/assets/styles/abstracts/variables';
```

Please use theme file (inside `libs/ui`) or duplicate code.

```suggestion
@import '@/assets/styles/abstracts/variables';
```

---

removed